src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -318,7 +318,7 @@ internal interface ISyntaxFactsService : ILanguageService
         string GetNameForArgument(SyntaxNode argument);
 
         bool IsNameOfSubpatternElement(SyntaxNode node);
-        bool IsPropertySubpattern(SyntaxNode node);
+        bool IsPropertyPatternClause(SyntaxNode node);
```

Comment:
Now i'm confused.  Are we calling them subpatterns, or pattern clauses?   :)   (My preference would be hte latter).

Reply:
Subpatterns are the things inside the clauses.